src/main/java/com/google/sps/servlets/EditBusinessBlobstoreServlet.java

```diff
@@ -0,0 +1,26 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.blobstore.BlobstoreService;
+import com.google.appengine.api.blobstore.BlobstoreServiceFactory;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpServletRequest;
+
+@WebServlet("/edit-business-blobstore")
+public final class EditBusinessBlobstoreServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    
+    BlobstoreService blobstoreService = BlobstoreServiceFactory.getBlobstoreService();
+    String uploadUrl = blobstoreService.createUploadUrl("/edit-business-data");
```

Comment:
I think this is giving me an error. I'll fix it in the next PR. This doesn't affect anyone else's features.

Reply:
oh it's actually ok! i closed the server temporarily and forgot and it threw an error and i thought it was this